src/main/java/de/retest/recheck/ui/diff/Warning.java

```diff
@@ -0,0 +1,60 @@
+package de.retest.recheck.ui.diff;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+import lombok.Data;
+
+/**
+ * The class Warning stores structurally the information about a test breakage. This information can be given in the
+ * review-GUI and recheck.cli, when someone applies a change.
+ */
+@Data
+public class Warning implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	@XmlAttribute
+	private final String elementIdentifier;
+
+	@XmlAttribute
+	private final String byMethodName;
```

Comment:
Hmmm, this couples recheck to the Selenium `By` mechanism. But regardless of the `By`, fields like `elementIdentifier` suggest that this is a warning specifically for the GUI element identification problem?

Reply:
@roesslerj what's your say on this, because I tend to generalize it here (without losing information). Maybe simply some more generic but descriptive fields.